themes/doom-henna-theme.el

```diff
@@ -35,42 +35,43 @@ determine the exact padding."
   ;; name        default   256       16
   ((bg         '("#21272e" nil       nil            ))
    (bg-alt     '("#21242b" nil       nil            ))
-   (base0      '("#1B2229" "black"   "black"        ))
-   (base1      '("#181A1F" "#1e1e1e" "brightblack"  ))
-   (base2      '("#202328" "#2e2e2e" "brightblack"  ))
-   (base3      '("#23272e" "#262626" "brightblack"  ))
-   (base4      '("#606F73" "#3f3f3f" "brightblack"  ))
-   (base5      '("#5B6268" "#525252" "brightblack"  ))
-   (base6      '("#73797e" "#6b6b6b" "brightblack"  ))
-   (base7      '("#9ca0a4" "#979797" "brightblack"  ))
-   (base8      '("#DFDFDF" "#dfdfdf" "white"        ))
+   (base0      '("#10151a" "black"   "black"        ))
+   (base1      '("#181A1F" "#2e2e2e" "brightblack"  ))
+   (base2      '("#1B1F23" "#262626" "brightblack"  ))
+   (base3      '("#262D35" "#3f3f3f" "brightblack"  ))
+   (base4      '("#282C34" "#525252" "brightblack"  ))
+   (base5      '("#2c313a" "#6b6b6b" "brightblack"  ))
+   (base6      '("#3B4048" "#979797" "brightblack"  ))
+   (base7      '("#495162" "#dfdfdf" "white"        ))
+   (base8      '("#606F73" "#1e1e1e" "brightblack"  ))
    (fg         '("#f8f8f8" "#bfbfbf" "brightwhite"  ))
-   (fg-alt     '("#737c8c" "#979797" "white"        ))
-   (grey       base4)
+   (fg-alt     '("#6B717D" "#979797" "white"        ))
+   (grey       '("#737c8c"))
    (red        '("#e74c3c" "#ff6655" "red"          ))
    (green      '("#53df83" "#99bb66" "green"        ))
-   (teal       '("#1abc9c" "#44b9b1" "brightgreen"))
+   (teal       '("#1abc9c" "#44b9b1" "brightgreen"  ))
    (blue       '("#56b5c2" "#51afef" "brightblue"   ))
    (cyan       '("#56b6c2" "#46D9FF" "brightcyan"   ))
 
-   ;; Not used
-   (orange     '("#da8548" "#dd8844" "brightred"    ))
+   ;; Not used, so remap to other (henna) colors
+   (orange     '("#9cd230" "#dd8844" "brightred"    ))
    (yellow     '("#ECBE7B" "#ECBE7B" "yellow"       ))
-   (magenta    '("#c678dd" "#c678dd" "brightmagenta"))
-   (violet     '("#a9a1e1" "#a9a1e1" "magenta"      ))
-   (dark-blue  '("#2257A0" "#2257A0" "blue"         ))
+   (magenta    '("#30c965"))
+   (violet     fg-alt)
```

Comment:
The main reason is that since this theme doesn't have purple/pink(ish) colors, I decided to switch them to a commonly used colors in the theme. Initially I had both as `fg` as a means to show elements as if they were un-styled. I later switched magenta to green just to try things out, and at the end of the day it was just a guess. Ill find some good colors, as you suggested, to be used for those two colors.